src/dict.c

```diff
@@ -329,9 +331,13 @@ int dictRehash(dict *d, int n) {
     unsigned long s0 = DICTHT_SIZE(d->ht_size_exp[0]);
     unsigned long s1 = DICTHT_SIZE(d->ht_size_exp[1]);
     if (dict_can_resize == DICT_RESIZE_FORBID || !dictIsRehashing(d)) return 0;
+    /* If dict_can_resize is DICT_RESIZE_AVOID, we want to avoid rehashing. 
+     * We can restrict the condition because both s0 and s1 are powers of 2.
+     * - If expanding, the _dictNextExp of dict_force_resize_ratio is 8.
+     * - If shrinking, the _dictNextExp of (HASHTABLE_MIN_FILL / dict_force_resize_ratio) is 1/32. */
```

Comment:
I've just been trying to understand why it's 32, not 64, and I thinks that others will have that understanding cost if we don't make both sides consistent.